packages/react/.oxlintrc.json

```diff
@@ -0,0 +1,1261 @@
+{
+  "$schema": "./node_modules/oxlint/configuration_schema.json",
+  "plugins": [],
+  "categories": {
+    "correctness": "off"
+  },
+  "env": {
+    "builtin": true
+  },
+  "jsPlugins": ["eslint-plugin-react-refresh"],
+  "overrides": [
+    {
+      "files": ["**/*.{js,jsx,ts,tsx}"],
+      "rules": {
+        "no-unused-vars": [
+          "error",
+          {
+            "vars": "all",
+            "varsIgnorePattern": "^_",
+            "args": "after-used",
+            "argsIgnorePattern": "^_"
+          }
+        ],
+        "react/react-in-jsx-scope": "off",
+        "react/prop-types": "off",
+        "react/display-name": ["error", {}]
+      },
+      "globals": {
```

Comment:
FWIW you can probably replace this huge list by just adding `browser` or `node` to the env option, depending on the application.

I thought I had fixed this problem in the @oxlint/migrate script a while ago, but maybe not 🤔